scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -194,16 +199,17 @@ class MicrobeEditor{
         }
 
         newName = playerSpecies.genus + " " + playerSpecies.epithet;
-        membrane = playerSpecies.speciesMembraneType;
+        rigidity = playerSpecies.membraneRigidity;
         colour = playerSpecies.colour;
         GenericEvent@ event = GenericEvent("MicrobeEditorActivated");
         NamedVars@ vars = event.GetNamedVars();
 
         vars.AddValue(ScriptSafeVariableBlock("name", newName));
-        vars.AddValue(ScriptSafeVariableBlock("membrane", membraneTypeToString(membrane)));
+        vars.AddValue(ScriptSafeVariableBlock("membrane", SimulationParameters::membraneRegistry().getInternalName(membrane)));
         vars.AddValue(ScriptSafeVariableBlock("colourR", colour.X));
         vars.AddValue(ScriptSafeVariableBlock("colourG", colour.Y));
         vars.AddValue(ScriptSafeVariableBlock("colourB", colour.Z));
+        vars.AddValue(ScriptSafeVariableBlock("rigidity", rigidity * 10));
```

Comment:
Why is rigidity multiplied by 10 here? any adjustment for display should be done in the GUI and not on this side.